nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/scheduling/TestProcessorLifecycle.java

```diff
@@ -191,6 +193,7 @@ public void validateStopCallsAreMeaninglessIfProcessorNotStarted() throws Except
      * operations can only be @OnScheduled, @OnUnscheduled, @OnStopped.
      */
     @Test
+    @Category(IntegrationTest.class)
```

Comment:
Forgot to mention. The two tests that you see here (as an example) were failing sporadically, due to unexpected delays related to the load of the full build.